src/Persistence_matrix/include/gudhi/Persistence_matrix/base_pairing.h

```diff
@@ -216,13 +213,21 @@ inline void Base_pairing<Master_matrix>::_remove_last(pos_index columnIndex)
       deathToBar_.erase(it);
     };
   }
+
+  auto it = RUM::map_.find(columnIndex);
+  if (it != RUM::map_.end()){
+    idToPosition_.erase(it->second);
+    RUM::map_.erase(it);
+  }
 }
 
 template <class Master_matrix>
 inline Base_pairing<Master_matrix>& Base_pairing<Master_matrix>::operator=(Base_pairing<Master_matrix> other) 
```

Comment:
The comment for copy/move construction implied removing this operator as well, since having a user-declared assignment operator disables the implicit generation of a move constructor (and deprecates the copy).
If you think the default `operator=` doesn't do the right thing (calling `operator=` on each base and member), then we better bring back the constructors (possibly with `=default;` so they aren't too long).

Reply:
Sorry, I just forgot it.